README.md

```diff
@@ -56,5 +56,72 @@ console.log(errors.AccessDenied);
 
 ```
 
+## Clustering
```

Comment:
It could be a good idea to have the list of public method no ?
You know, something similar to nodejs documentation